bower.json

```diff
@@ -22,12 +22,12 @@
     "wct.conf.js"
   ],
   "dependencies": {
-    "polymer": "Polymer/polymer#^2.0.0-rc.1"
+    "polymer": "Polymer/polymer#^v2.0.0"
```

Comment:
What's the reason of adding `v`? There's no `v` in the `bower.json` standard: https://github.com/bower/spec/blob/master/json.md